tlm-stencil/src/components/mw-checkbox-button/mw-checkbox-button.tsx

```diff
@@ -0,0 +1,37 @@
+import {Component, getAssetPath, h, Prop} from '@stencil/core';
+
+@Component({
+  tag: 'mw-checkbox-button',
+  styleUrl: 'mw-checkbox-button.css',
+  assetsDirs: ['../assets'],
+  shadow: false,
+})
+export class MwCheckboxButton {
```

Comment:
I think this should not be called "Button". So let's just change all occurances to `mw-checkbox`